src/app/[practiceCode]/book/page.tsx

```diff
@@ -25,9 +26,25 @@ export default async function Appointment({
 
   const [{ id, name, logo, street_address, city, phone, website }] =
     data as Practice[];
-
+  const ogImage = logo || '/connectient-logo.png';
+  const ogMetadata = {
+    title: name,
```

Comment:
@NuriyaAkh The practice title is coming through, which is great, but not the practice logo. Did you get this in your testing?

Reply:
I see open graph images, my pr ready for review